packages/grafana-ui/src/components/ButtonCascader/ButtonCascader.story.tsx

```diff
@@ -1,20 +1,24 @@
 import React from 'react';
-import { withKnobs, text, boolean, object, select } from '@storybook/addon-knobs';
+import { Story } from '@storybook/react';
 import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
 import { ButtonCascader } from '@grafana/ui';
+import { NOOP_CONTROL } from '@grafana/ui/.storybook/preview';
+import { ButtonCascaderProps } from './ButtonCascader';
 
 export default {
   title: 'Forms/Cascader/ButtonCascader',
   component: ButtonCascader,
-  decorators: [withKnobs, withCenteredStory],
-};
-
-const getKnobs = () => {
-  return {
-    disabled: boolean('Disabled', false),
-    text: text('Button Text', 'Click me!'),
-    icon: select('Icon', ['plus', 'minus', 'table'], 'plus'),
-    options: object('Options', [
+  decorators: [withCenteredStory],
+  parameters: {
+    docs: {},
```

Comment:
Is this empty object needed?

Reply:
I don't think so.
Would remove it right away.

Thanks for bringing this to my atention @jackw ! 🙌